debian/install-docker.sh

```diff
@@ -0,0 +1,20 @@
+echo "Installing Docker..."
+apt-get update
+apt-get install apt-transport-https ca-certificates curl gnupg2 software-properties-common
+
+curl -fsSL https://download.docker.com/linux/debian/gpg | sudo apt-key add -
+
+apt-key fingerprint 0EBFCD88
+
+add-apt-repository    "deb [arch=amd64] https://download.docker.com/linux/debian \
```

Comment:
Need to change /linux/debian to /linux/ubuntu.

Reply:
Just this line, above Debian key retrieval works fine.

Reply:
This is in the `debian/` subdirectory, and I've only tested it against Debian. I can:
- make a separate directory and flesh out the equivalent Ubuntu scripts
- OR just overhaul the whole thing to be only Ubuntu.

I was assuming this would be the first step towards having separate bootstrap scripts for various platforms (e.g. Debian, Ubuntu, CentOS, CoreOS, etc.) Thoughts?

Reply:
Good catch! 19f2af1

Reply:
Created a separate directory for Ubuntu in 1d06de4.

Reply:
Debian is fine with me.  I went straight to Ubuntu and didn't read the docs well enough :-)

Reply:
My mistake- I went straight to Ubuntu.  I'll test with Debian.